documentation/api/io.strimzi.api.kafka.model.listener.KafkaListenerAuthenticationCustom.adoc

```diff
@@ -29,41 +29,43 @@ spec:
             - name: example
 ----
 
-.Protocol map
-A protocol map is generated that uses the above `sasl` and `tls` fields to determine which protocol to map to the listener.
+A protocol map is generated that uses the `sasl` and `tls` values to determine which protocol to map to the listener.
 
 * SASL = True, TLS = True -> SASL_SSL
 * SASL = False, TLS = True -> SSL
 * SASL = True, TLS = False -> SASL_PLAINTEXT
 * SASL = False, TLS = False -> PLAINTEXT
 
-.`listener-config`
-All configuration listed under `listener-config` is prefixed with `listener.name._<listener_name>-<port>_`.
+=== `listenerConfig`
+Listener configuration specified using `listenerConfig` is prefixed with `listener.name._<listener_name>-<port>_`.
 For example, `sasl.enabled.mechanisms` becomes `listener.name._<listener_name>-<port>_.sasl.enabled.mechanisms`.
 
-.Secrets
-`Secrets` are mounted to `/opt/kafka/custom-authn-secrets/custom-listener-_<listener_name>-<port>_/_<secret_name>_` in the Kafka broker nodes' containers.
+=== `secrets`
+Secrets are mounted to `/opt/kafka/custom-authn-secrets/custom-listener-_<listener_name>-<port>_/_<secret_name>_` in the Kafka broker nodes' containers.
 
-For example, the mounted secret (`example`) in the example configuration would be located at `/opt/kafka/custom-authn-secrets/custom-oauth-bespoke-9093/example`.
+For example, the mounted secret (`example`) in the example configuration would be located at `/opt/kafka/custom-authn-secrets/custom-listener-oauth-bespoke-9093/example`.
 
-.Principal
-You can set a custom principal builder in the Kafka cluster config.
+=== Principal builder
+Kafka's principal builder supports authentication.
+You can implement a custom authentication principal builder in the Kafka cluster configuration.
```

Comment:
I don't think this is really right. The principal builder does not do authentication. It just builds the principal. I think the original statement was better -> or at least the first sentence here is not really making much sense to me. Also, the principal builder is cluster wide. So it is not _custom authentication principal builder_. It is _custom principal builder_,